### DISCUSSION TITLE
Adding to VS Code

### DISCUSSION THREAD

**Discussion by smiller0924** [BinaryKits/ZPLUtility#155] [Q&A] [answered]
I have added the NuGet Package Manager to my VS Code.  How do I add this package to my VS Code environment.

Thank you.

**Comment by primo-ppcg** [comment, marked as answer]
Assuming you've already created a project:

1. From the menu strip select _Terminal_ -> _New Terminal_
2. Ensure that the prompt path is the project directory, if not navigate there via `cd`.
3. Type: `dotnet add . package BinaryKits.Zpl.Viewer` (or which ever package is needed).
